client/me/help/help-contact/index.jsx

```diff
@@ -436,7 +438,11 @@ const HelpContact = React.createClass( {
 
 	shouldUseDirectly: function() {
 		const isEn = this.props.currentUserLocale === 'en';
-		return config.isEnabled( 'help/directly' ) && isEn;
+		return (
+			config.isEnabled( 'help/directly' ) &&
+			isEn &&
+			this.props.directlyStatus !== DIRECTLY_STATUS_ERROR
```

Comment:
Maybe we could create a selector for this called `isDirectlyFailed` that way we don't need to add case logic for the status in our component.